jsapp/scss/components/_kobo.connect-projects.scss

```diff
@@ -0,0 +1,208 @@
+
+      .kobo-select__wrapper {
+        width: 50%;
+        margin-right: 50px;
+        label {
+          margin-top: 0px;
+        }
+        .kobo-select__placeholder {
+          color: $kobo-black;
+        }
+      }
+
+      .text-box {
+        width: 35%;
+        margin-top: 0px;
+        margin-right: 24px;
+        input {
+          padding: 9px 10px;
+          background-color: $kobo-white;
+          color: $kobo-black;
+        }
+      }
+
+      .kobo-button {
+        max-height: 40px;
+      }
+    }
+  }
+  .connect-projects__import--list {
+    margin-top: 20px;
+
+    label {
+      margin-top: 20px;
+      font-size: 14px;
+      font-weight: bold;
+      color: $kobo-darkgray;
+    }
+
+    .connect-projects__import--list-item {
+      position: relative;
+      display: flex;
+      justify-content: space-between;
+      padding-bottom: 10px;
+      margin-top: 7px;
+      margin-bottom: 10px;
+      border-bottom: 1px solid;
+      border-color: $kobo-lightgray;
+
+      i.k-icon-check {
+        color: $kobo-blue;
+      }
+
+      .connect-projects__import--labels {
+        position: absolute;
+        top: 6px;
+        left: 32px;
+        font-weight: 500;
+
+        i.k-icon-check {
+          color: $kobo-blue;
+          font-weight: bold;
+        }
+
+        .connect-projects__import--labels--source {
+          margin-left: 24px;
+          font-weight: 400;
+          color: $kobo-darkgray;
+        }
+      }
+
+      .connect-projects__import--options {
+        position: relative;
+        button {
+          position: absolute;
+          right: 0;
+          padding-bottom: 25px;
+
+          i.k-icon {
+            font-size: 24px;
+          }
+        }
+
+        button:not(:first-child) {
+          margin-right: 45px;
+        }
+      }
+    }
+    .loading__inner {
+      i {
+        vertical-align: text-bottom;
+      }
+    }
+
+    .connect-projects__no-imports {
+      @extend .connect-projects__import--list-item;
+      font-style: italic;
+      color: $kobo-midgray;
+      padding-bottom: 10px;
+    }
+  }
+}
+
+// Compensate for when sidebar(s) messes up modal a bit
+@media
+  (min-width: 1000px) and (max-width: 1140px),
+  (min-width: 770px) and (max-width: 860px),
+  (max-width: 700px) {
+  .connect-projects__export--multicheckbox {
+    display: block !important;
+
+    .multi-checkbox {
+      margin-top: 12px;
+      width: 100% !important;
+      overflow-x: scroll;
+    }
+  }
+
+  .connect-projects__import--form {
+    display: block !important;
+
+    .kobo-select__wrapper {
+      width: 100% !important;
+      margin-bottom: 12px;
+    }
+
+    .text-box {
+      width: 100%;
+    }
+
+    .kobo-button {
+      display: block;
+      margin-top: 12px;
+      width: 70%;
+      margin-left: auto;
+      margin-right: auto;
+    }
+  }
+}
+
+@media screen and (max-width: 530px) {
+  .connect-projects__export--options {
+    display: block !important;
```

Comment:
Similarly to above :) I would probably just drop the `530px` styles and instead try having `flex-wrap: wrap` on the `connect-projects__export-row` :)